x-pack/test/security_solution_cypress/es_archives/risk_hosts_updated/mappings.json

```diff
@@ -31,10 +31,6 @@
     },
     "settings": {
       "index": {
-        "lifecycle": {
```

Comment:
Can you please provide an explanation for the issue that caused the test failures, and why removing these ILM settings fixes them? Other developers (myself included; this one was tricky) would benefit greatly from your findings here.

Reply:
My understanding, is that ILM is not supported in serverless, which mean they not supported in cypress tests, I put some related PR/error in description

Reply:
Another idea, that maybe it was partially fixed by this [PR](https://github.com/elastic/kibana/pull/173024), when EA tests were separated to a different execution

Reply:
@nkhristinin it looks like link you provided to the error has expired. Can you please post something more permanent here for posterity? That way if this test fails again, we can validate whether the hypothesized ILM error is to blame.

Reply:
RE the ILM hypothesis: in [my investigation](https://github.com/elastic/kibana/pull/169714#issuecomment-1823012514) I saw failures in ESS as well, so I think that invalidates ILM as the underlying issue.

I also don't think ILM explains the behavior I was observing: the test fails due to the _presence_ of unexpected data (specifically, historical risk scores).

Reply:
Not sure, what was with the link, but I changed it to the [link](https://buildkite.com/elastic/kibana-pull-request/builds/179121) of failing build in this pr.

The serverless tests are failing because of ILM, I am not sure if it's related to your initial investigation, or if it's a new problem.

Reply:
I had not encountered that ILM error during my investigation, and since it's specific to serverless it cannot be the root cause of https://github.com/elastic/kibana/issues/169154.

Reply:
Yes, it's not the root cause of those issues. After more deep look into that, this is my understanding of the timeline:

We did have some flaky tests, which you investigate in this [PR](https://github.com/elastic/kibana/pull/169714)

After some time, EA tests were [moved](https://github.com/elastic/kibana/pull/173024) to separate executions

When I started to work on those tests, there [were](https://buildkite.com/elastic/kibana-pull-request/builds/179121) [constant](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/4062) test failures in serverless, not flaky tests, which this PR if fixed.

Then I ran several flaky test runs, which were successful for enrichments, but had failures not related to this test case:

[x100](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/4673#018c91cf-b427-4d30-90bc-9daa2b29ca5b) - errors related to dashboard

[x200](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/4679#018caa34-a9c0-4473-a2ce-50dd6799a289) - errors related to dashboard

And then I saw that there was one more [x200](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/4726#018cce75-f705-4d47-a81d-ad3d431ee995) test run, which had 1 error (test 175) for enrichment tests, but this error also not related to original issues, but something that happened in `before all` hook

Original error:
>AssertionError: Timed out retrying after 150000ms: Expected not to find content: 'Original host risk level' within the element: <div.euiFlexGroup.StyledEuiFlexGroup-sc-1az7ye2-2.chTqHU.css-1a15qtm-euiFlexGroup-responsive-none-flexStart-center-row> but continuously found it.
    at Context.eval (webpack:///./e2e/entity_analytics/enrichments.cy.ts:129:69)
    
 New error:
    
 
> CypressError: `cy.task('esArchiverUnload')` timed out after waiting `60000ms`.
 https://on.cypress.io/api/task
 Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Enrichment`

Conclusion: 
We have not yet seen that original issue, but there were other flaky tests, so I will continue to run the flaky test runner, and we should have more details.

Reply:
No, the last test run confirmed that it still [flaky](https://buildkiteartifacts.com/e0f3970e-3a75-4621-919f-e6c773e2bb12/fedcad1e-0f36-4d55-bdc7-e1edbecf91fe/018ce86c-5812-4d34-aa2e-cb157c520637/018ce86d-16ed-4a91-beb2-333794ef28ea/target/test_failures/018ce86d-16ed-4a91-beb2-333794ef28ea_9ba7a20577e24995f8bf715d4d8332ca.html?response-content-type=text%2Fhtml&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQPCP3C7LQXHEAG2A%2F20240108%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240108T154132Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEDwaCXVzLWVhc3QtMSJGMEQCIBCKzv7cKtRrG0kmri2r4lTEuxy7HyeaIY3GBvUE%2FKYEAiBhSrQUBlDPltmC21y5PLa0IEAvy%2FdrrTBdk2Fo0WN7eyr6AwjV%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F8BEAAaDDAzMjM3OTcwNTMwMyIMfw4blaOTp19ITxBoKs4Dj8kxW4QgOj5kivh4rCd92nHFB4HwuWUFzMCSU7XjW3gf%2BHYEKXQlzvII7Djo%2F8Lt5OtX50u%2FVh9wXz7OlPj8ytbcSoi6%2F%2FTH8InLWPQYkVs0wJfXcv2d%2FOkmgzyOlBRaETyRIMhILZPkTcXS%2FMdrTh6q10bwwn9XBW4%2BC1m9Ba%2FZiXLiA1LlHBxMGG%2FYjFkrNYzSf72gibolIkB8f8kACBvITR5dghcHi%2Fc3vrVcOFLx8kHTpxjkfxZafj4Ngni2gouEfOcYxruWXFGCnZhawijNj8Imh1KF92OHcuS2HA%2Fcv07yZGCLklbYza0iMnzOLND6MBRZCCEQh13wvN9vJ%2BwZ0xLqG8y5v3SRRSkMAaTeFtT80viJ4c%2BMnkex45yqEEqudl%2B%2BOgV0yjHs3%2BvXqLJCNYvvyPBZyT6m1Ipc03XAvieRNl%2FXDYBYEggNr3dSk0dXwP%2B6v9YzzdyHh0QCG9Sk72bONy58BZQ72Km38FVCRFvIbOh8qnm%2FBnw%2BdrgqyW%2B2MQeyKc058ld0aVCkF5V85OE1JtXxpoyd2nu7uoRbJ2YpCtodVd%2FfEO%2FAliBp4UF8bsdBN9hRH26%2FsgP6RJxM9BUFLaNpEc1fnOCYMJW%2B76wGOqYBXjso%2BpaDWeDSxQ9hr7fSz7ToQP6%2FWHuEtcqb884ar%2BMj1kehBr%2B%2BzjIpD7eeN3Iz73DAsjwM1mQewyB3nOoUFr9aAe6yhjwxrxbkBmlgLf7ANUeRk4nKgc9MnMRuta0JLHT9g7EblcmXHjBezPPklAkF217nS7JA0gECJFFjqtNKCsMtLgXM1QsP3ryB%2FprR8O7%2B38Mah7Z6%2FqWL0dJoUq513c09NA%3D%3D&X-Amz-SignedHeaders=host&X-Amz-Signature=7e349d205a0646f52d2527d914f222808ec75b53ecf3e3c6d0ac943822a19c50) -